runtime/queries/zig/highlights.scm

```diff
@@ -0,0 +1,277 @@
+[
+  (container_doc_comment)
+  (doc_comment)
+  (line_comment)
+] @comment
+
+(IDENTIFIER) @variable
```

Comment:
Does highlighting really work? I was under the assumption that it would highlighting everything as a variable at this point (since I don't think you can override them). I might be wrong since `locals.scm` is involved.

Reply:
No, it does not, You are right.